### PR TITLE
I think linking is enough

### PR DESCRIPTION
**Pull request by plaindocs**
@mjang @Bradamant3 @nbkaplan @mxsasha 

I faffed around on how to link or include Mike's PR #1150 and settling on this. Any objections?

### CODE REVIEW

File: docs/conf/portland/2020/example-proposal.rst [hunk 1]
```diff
@@ -1,14 +1,16 @@
-If you need guidance on how to create a proposal, read this page.
+:template: {{year}}/generic.html
+
+Example talk proposal
```
Comment (mjang) [inline docs/conf/portland/2020/example-proposal.rst]:
```suggestion
Example talk proposal title
====================
```

File: docs/conf/portland/2020/example-proposal.rst [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+=====================
```
Comment (mjang) [inline docs/conf/portland/2020/example-proposal.rst]:
```suggestion
Example talk proposal title
===========================
```

**Review by mjang — APPROVED** [review]
Thanks @plaindocs ! One suggestion (approving as I think the PR's still fine otherwise).

File: docs/conf/prague/2020/example-proposal.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
+:template: {{year}}/generic.html
+
+Example talk proposal
+=====================
```
Comment (plaindocs) [inline docs/conf/prague/2020/example-proposal.rst]:
```suggestion
Example talk proposal title
===========================
```

**Pull request merged**